### DISCUSSION TITLE
Date of messages

### DISCUSSION THREAD

**Discussion by abcpop** [valinet/ExplorerPatcher#424] [General]
Small bug ?
All messages shows 29.8 as date
https://ibb.co/THHYFzX

**Comment by valinet** [comment]
Yeah, it's hardcoded at that. Did not find a convenience function to format the date into the string the notifications API uses when I implemented that, haven't looked too much either, so I hardcoded it to the date I wrote it and called it a day. Not a big deal, could see an improvement some day or left like that, less likely to intruduce other issues.
